### DISCUSSION TITLE
Resource for setup R in vscode

### DISCUSSION THREAD

**Discussion by sahaavi** [REditorSupport/vscode-R#1329] [General]
https://fanwangecon.github.io/R4Econ/development/install/htmlpdfr/fs_install_R.pdf

**Comment by SondraKohlesAPh** [comment]
This link appears to be a resource for installing R on various operating systems, including Windows, macOS, and Linux. The resource provides step-by-step instructions for downloading and installing R, as well as the RStudio integrated development environment. It also covers the installation of various R packages, including the tidyverse, ggplot2, and dplyr, which are commonly used in data analysis and visualization.

The resource is part of the R for Economists (R4Econ) project, which aims to provide learning resources and tools for economists who want to use R for data analysis and econometrics. The project includes various courses, books, and online resources, and is maintained by Fan Wang, an economist at the University of Houston.
